analyses/cms-open-data-ttbar/helper.cpp

```diff
@@ -31,10 +31,8 @@ TH1D Slice(TH1D &h, double low_edge, double high_edge)
 }
 
 // functions creating systematic variations
-inline TRandom &get_thread_local_trandom()
-{
-   thread_local TRandom rng;
-   rng.SetSeed(gRandom->Integer(1000));
+static TRandom &get_thread_local_trandom() {
```

Comment:
I don't know whether the intention was to say that this function returns a reference to a `static TRandom`, but I'm pretty sure what it really says is that this function has `static storage duration` rather than being `inline`.

So this change from `inline` to `static` here made me study again the difference between the two for specifiers. And I'm probably missing something: I don't understand why we have either. Why not just:

```suggestion
TRandom &get_thread_local_trandom() {
```
?